src/provider_completions.jl

```diff
@@ -1,7 +1,23 @@
 function process(r::JSONRPC.Request{Val{Symbol("textDocument/completion")},TextDocumentPositionParams}, server)
     tdpp = r.params
     line = get_line(tdpp, server)
-    comp = Base.REPLCompletions.completions(line, chr2ind(line,tdpp.position.character))[1]
+    io = IOBuffer(reverse(line[1:chr2ind(line,tdpp.position.character)]))
```

Comment:
Why use an ``IOBuffer`` here? Why not just iterate over the characters of the string via ``for c in s`` instead of the ``while`` loop later?